io_scene_godot/converters/animation/animation_data.py

```diff
@@ -175,6 +180,17 @@ def export_stashed_track(self, escn_file, export_settings, stashed_track):
                             ActionStrip(strip),
                             anim_resource
                         )
+                        self.clear_action_effect()
+
+    def clear_action_effect(self):
+        """Clear side effect of exporting an action"""
+        if (isinstance(self.blender_object, bpy.types.Object) and
+                self.blender_object.pose is not None):
+            bpy.context.scene.objects.active = self.blender_object
+            bpy.ops.object.mode_set(mode='POSE', toggle=False)
+            bpy.ops.pose.select_all(action='SELECT')
```

Comment:
This only works if all bones are unhidden. Try hiding the upper body bones in the example from #199, and you'll get the same result as before the patch.

This is still better than nothing though.

Reply:
Something like https://blender.stackexchange.com/a/79507/60829 might work. I gave this a try and it seems to work without requiring bones to be unhidden:

```
    def clear_action_effect(self):
        """Clear side effect of exporting an action"""
        if (isinstance(self.blender_object, bpy.types.Object) and
                self.blender_object.pose is not None):

            for bone in self.blender_object.pose.bones:
                bone.rotation_quaternion = mathutils.Quaternion((0, 0, 0), 0)
                bone.scale = mathutils.Vector((1, 1, 1))
                bone.location = bone.bone.head
```

Reply:
Scratch that, it completely messed up the exported animations :D

Reply:
Hmm..  maybe simply toggle every bone to unhidden in exporting and then toggle them back,
let me give a try

Reply:
well, I think your approach is right, just set the pose matrix to identity would work.